packages/plugin-hardhat/src/utils/simulate-deploy.ts

```diff
@@ -51,7 +50,7 @@ async function getSimulatedData(
   const deployData = await getDeployData(hre, ImplFactory, opts);
   const simulateDeploy = async () => {
     return {
-      abi: ImplFactory.interface.format(FormatTypes.minimal) as string[],
+      abi: ImplFactory.interface.format(true) as string[],
```

Comment:
Casting seems no longer necessary.
```suggestion
      abi: ImplFactory.interface.format(true),
```